source/demo/Wizard/Wizard.js

```diff
@@ -31,6 +32,9 @@ export default class Wizard extends Component {
     };
   }
 
+  updateState = obj =>
+    this.setState(state => ({ ...state, ...obj, key: state.key + 1 }));
```

Comment:
Oh, thanks, I didn't know that setState functional callback will also merge with previous state. I'll update the code.